client/configuration_variable_test.go

```diff
@@ -291,3 +295,37 @@ var _ = Describe("Configuration Variable", func() {
 		})
 	})
 })
+
+func TestConfigurationVariableMarshelling(t *testing.T) {
```

Comment:
this test was added to verify that the marshal and unmarshal are working as expected.